src/main/resources/validationrules/prm/prm-role-mapping-validation.drl

```diff
@@ -0,0 +1,39 @@
+package validationrules.core;
+import uk.gov.hmcts.reform.roleassignment.domain.model.RoleAssignment;
+import uk.gov.hmcts.reform.roleassignment.domain.model.Request;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.Status;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RequestType;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleCategory;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleType;
+import function uk.gov.hmcts.reform.roleassignment.domain.service.common.ValidationModelService.logMsg;
+
+rule "prm_create_org_role"
```

Comment:
Please add comment before rule similar to other ORM comment https://github.com/hmcts/am-role-assignment-service/blob/88dcba9706fd3467b5b91ea6bcaec724fd933441/src/main/resources/validationrules/core/organisational-role-mapping-common.drl#L11-L17

Reply:
added comments about create rule on commit [f8269a9](https://github.com/hmcts/am-role-assignment-service/pull/2062/commits/f8269a94896e32c2d336b6c76aa7ff585d45c1b5)